sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeBusinessCardsOptions.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+
+/**
+ * Options that may be passed when using recognize business card APIs on Form Recognizer client.
+ */
+@Fluent
+public final class RecognizeBusinessCardsOptions {
+    private static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(5);
```

Comment:
Something to consider, I see this in all `Options` classes.  Any reason why this is not set somewhere general int he library so the options can reference that value instead?

Reply:
it can be moved to Utility class. Thanks Mariana